account/urls.py

```diff
@@ -0,0 +1,66 @@
+from django.contrib.auth import views as auth_views
+from django.urls import path
+
+urlpatterns = [
+    path(
+        'login/',
+        auth_views.LoginView.as_view(template_name='account/login.html'),
+        name='login',
+    ),
+
+    path(
+        'logout/',
+        auth_views.LogoutView.as_view(template_name='account/logged_out.html'),
```

Comment:
When I click on `logout`, I get redirected to the home: I never see the `logged_out` template.

Either the config should be changed or the template removed :-)

Reply:
Mhm, no. Every page needs a `template_name`, otherwise the default value is used (which doesn't point to any valid template file and would break the view if `LOGOUT_REDIRECT_URL` was removed from `settings.py`). So I'd keep it like this and prepare a custom `logged_out.html`.